packages/main/src/ComboBox.js

```diff
@@ -311,13 +325,17 @@ class ComboBox extends UI5Element {
 
 	_afterClosePopover() {
 		this._iconPressed = false;
+
+		if (isPhone()) {
+			this.blur();
```

Comment:
what is the reason for this, otherwise it is being opening again or?

Reply:
It is to close the device keyboard and prevent focus to stay in the ui5-combobox, so that the user can't write in it.